install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -99,11 +99,12 @@ spec:
             - name: STRIMZI_LEADER_ELECTION_LEASE_NAMESPACE
               valueFrom:
                 fieldRef:
-                  fieldPath: metadata.namespace
-            - name: STRIMZI_LEADER_ELECTION_IDENTITY
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
+                   fieldPath: metadata.namespace
+           - name: STRIMZI_LEADER_ELECTION_IDENTITY
+              spec:
+               valueFrom:
+                 fieldRef:
+                   fieldPath: metadata.name
```

Comment:
I think the env var declaration `STRIMZI_LEADER_ELECTION_IDENTITY` is not aligned correctly.

Reply:
Fixed